test/torchaudio_unittest/transforms/autograd_test_impl.py

```diff
@@ -276,10 +276,9 @@ def test_psd_with_mask(self, multi_mask):
 
     @parameterized.expand([
         "ref_channel",
-        # stv_power test time too long, comment for now
+        # stv_power and stv_evd test time too long, comment for now
         # "stv_power",
-        # stv_evd will fail since the eigenvalues are not distinct
-        # "stv_evd",
+        "stv_evd",
```

Comment:
should this be commented out per the comment above?

Reply:
Oh you are right lol. Not sure what I was doing here.